services/121-service/src/program-financial-service-provider-configurations/program-financial-service-provider-configurations.controller.ts

```diff
@@ -102,7 +102,7 @@ export class ProgramFinancialServiceProviderConfigurationsController {
   @AuthenticatedUser({ isAdmin: true })
   @ApiOperation({
     summary:
-      'Update a Financial Service Provider Configuration for a Program. Can only update the label and properties. Posting an empty array of properties will delete all properties. It is recommand to use this endpoint for adding properties /programs/{programId}/financial-service-provider-configurations/{name}/properties. Example of how to format properties can also be found there',
+      'Update a Financial Service Provider Configuration for a Program. Can only update the label and properties. Posting an array with properties or an empty array of properties will overwrite all properties. If you want to add properties it is therfore recommended to use this endpoint: /programs/{programId}/financial-service-provider-configurations/{name}/properties. Example of how to format properties can also be found there',
```

Comment:
instead of "overwrite all properties": "delete all existing properties and create new ones" to be more clear about what actually happens.